docker-bake.hcl

```
@@ -1,5 +1,5 @@
 variable "GO_VERSION" {
-  default = "1.21.10"
+  default = "1.22.8"
```

We could change that to 
```suggestion
  default = null
```

So it keeps default from the Dockerfile